### PR TITLE
Feature/24155 privacy page

### PR DESCRIPTION
**Pull request by danwilliams**

### CODE REVIEW

File: pupil-spa/src/app/privacy/privacy.component.html [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+        <a href="/ict-survey/privacy#content">Back to top</a>
+      </p>
+    </div>
+
+    <div class="column-one-third related">
+      <div class="border"></div>
+      <h2 class="heading-small heading-contents">Related</h2>
+      <a href="{{linkCookies}}">Cookies</a>
+    </div>
+
```
Comment (GeorgeChatzigiannis) [inline pupil-spa/src/app/privacy/privacy.component.html]: @danwilliams why soo much whitespace?

Comment (GuyHarwood) [reply pupil-spa/src/app/privacy/privacy.component.html]: easier to read IMO

Comment (GeorgeChatzigiannis) [reply pupil-spa/src/app/privacy/privacy.component.html]: @GuyHarwood I don't see the point for having 3 empty lines when 1 should be enough.
It's a non blocker really but just wanted to point it out.

Comment (danwilliams) [reply pupil-spa/src/app/privacy/privacy.component.html]: I've tweaked this slightly, comments were fair 👍

**Pull request merged**